libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -145,10 +145,16 @@ class AppPrompt {
     //   if (!e.target.closest(CONFIG.selectors.prompt)) AppPrompt.close();
     // });
 
+    document.addEventListener('keydown', this.handleKeyDown);
+
     [this.elements.closeIcon, this.elements.cta]
       .forEach((elem) => elem.addEventListener('click', this.close));
   };
 
+  handleKeyDown = (event) => {
+    if (event.key === 'Escape') this.close();
+  };
+
   initRedirect = () => setTimeout(() => {
     // TODO: uncomment for actual redirect
```

Comment:
Is this TODO still valid?

Reply:
For the time being, yes. But this is being merged in a feature branch, so it won't affect anything.

After a certain amount of time, a user would get redirected to another page; we want that behavior disabled during development, as we're interested in UI, data, accessibility, dismissal logic, etc. Another option would be to set the redirect time to a larger value, say 30 minutes, but we have plenty more TODOs in the code, so we'll make sure to address them all before opening a PR against `main`

Reply:
Yes, we still need it for the moment